integration-boilerplate/README.md

```diff
@@ -0,0 +1,12 @@
+# Wix Answers integration boilerplate
+In this boilerplate you will find the basic implantation in order to successfuly integrate with Wix Answers.
+The boilerplate is a node application written in typescript and it uses mongodb (easy to change) as an example of how we think integrations should be develope.
```

Comment:
develope -> developed

Reply:
The readme should also have instructions on how to run it locally (using the sandbox) and how to run tests